CONTRIBUTING.md

```diff
@@ -29,3 +29,78 @@ Recommendations and requirements for how to best contribute to Parquet. We striv
 ### License
 By contributing your code, you agree to license your contribution under the terms of the APLv2:
 https://github.com/apache/parquet-format/blob/master/LICENSE
+
+### Additions/Changes to the Format
+
+The general steps for adding features to the format are as follows:
+
+1. Discuss changes on on the developer mailing list (dev@parquet.apache.org).  Often times it is helpful to link to a draft pull request to make the discussion concrete. This step is complete when there lazy consensus.
+
+2. One a change has lazy consensus two implementations of the feature
+demonstrating interopability must also be provided.  One implementation MUST be [parquet-java](http://github.com/apache/parquet-java).  It is preferred that the second implementation be [parquet-cpp](https://github.com/apache/arrow) or [parquet-rs](https://github.com/apache/arrow-rs), however at the discretion of the PMC any
+open source Parquet implementation may be acceptable.
```

Comment:
Could extra consideration be given to implementations that maintain their column in the feature matrix?

Reply:
This sounds reasonable to me and I can update that as secondary option (ultimately this is the realm of the PMC)

Reply:
@etseidl I updated the language, please let me know if this looks reasonable.

Reply:
Thank you @emkornfield, the addition looks great to me.